cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -81,9 +81,15 @@ public void authorized(HttpRequest httpRequest, HttpResponder httpResponder) thr
       throw new BadRequestException("Missing request body");
     }
 
-    CheckAuthorizedResponse response = new CheckAuthorizedResponse(
-      auth.authorized(request.getEntity(), request.getUser(), request.getActions()));
-    httpResponder.sendJson(HttpResponseStatus.OK, response);
+    if (request.getActions() == null) {
```

Comment:
I think that's because of how GSON deserialization works, if the actions is not in the json.